src/ToDoListView.js

```diff
@@ -38,30 +44,53 @@ export default function ToDoListView() {
       const updatedToDos = [...toDoList, response.data];
       setToDoList(updatedToDos);
     });
-    handleClose();
+    handleAddClose();
   };
 
+  const handleEditOpen = (itemToEdit) => {
+    setEditItemTextBoxValue(itemToEdit.name);
+    setEditItemOpenFlag(true);
+  };
   const handleDelete = (itemToDeleteId) => {
     ToDoRequestService.deleteToDo(itemToDeleteId)
       // eslint-disable-next-line no-unused-vars
       .then(() => {
         setToDoList(toDoList.filter((toDo) => toDo.id !== itemToDeleteId));
       });
   };
+  const handleEditSave = () => {
+    ToDoRequestService.editToDo(editItemTextBoxValue);
+    //.then((response) => {
+    // const updatedToDos = toDoList.map((toDo) => {
+    //   if (toDo.id === response.data.id) {
+    //     return response.data;
+    //   }
+    //   return toDo;
+    // });
+    // setToDoList(updatedToDos);
+    //});
```

Comment:
What is this?

Reply:
Sorry I should've taken out these comments, it's to change the edited to do for the new one within the UI itself. I thought out the logic but I wasn't sure on what the patch would return so I just left it commented.